meshtastic/interfaces/ble/management_service.py

```
@@ -1,887 +1,27 @@
-                iface._finish_management_operation()
+__all__ = [
+    "BLEManagementCommandHandler",
+    "BLEManagementCommandsService",
+    "BLUETOOTHCTL_TRUST_TIMEOUT_SECONDS",
+    "TRUST_COMMAND_OUTPUT_MAX_CHARS",
+    "TRUST_HEX_BLOB_RE",
+    "TRUST_TOKEN_RE",
+    "_ManagementStartContext",
+    "_create_management_client",
+    "_is_blank_or_malformed_address_like",
+]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)